serverless/pages/clients-java-getting-started.asciidoc

```diff
@@ -96,7 +96,7 @@ ElasticsearchTransport transport = new RestClientTransport(
 ElasticsearchClient esClient = new ElasticsearchClient(transport);
 ----
 
-To get API keys or the Elasticsearch Endpoint for a project, see <<elasticsearch-get-started>>.
+To get API keys for the {es} Endpoint for a project, see <<elasticsearch-get-started>>.
```

Comment:
> To get API keys for the {es} Endpoint for a project, see <<elasticsearch-get-started>>.

Not at all the point of this PR, but I don't think endpoint should be capitalized. That'll be something for another cleanup some day.

Reply:
I did a lot of drive-by's in this PR, so one more won't hurt